js/index.js

```diff
@@ -40,3 +40,58 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+let navTag = document.querySelector('nav')
+let anchorTag = document.getElementsByTagName('a')
+let anchorArray = [siteContent["nav"]["nav-item-1"],siteContent["nav"]["nav-item-2"],siteContent["nav"]["nav-item-3"],
+siteContent["nav"]["nav-item-4"],siteContent["nav"]["nav-item-5"],siteContent["nav"]["nav-item-6"]];
+let madeAnchorArray = Array.from(anchorTag);
```

Comment:
consider using a loop since you already have an object like this: 
`for (let i = 0; i < navItems.length; i++) {
  navItems[i].textContent = siteContent.nav[`nav-item-${i + 1}`];`